workflow-service-sdk/api/openapi.yaml

```diff
@@ -127,6 +127,41 @@ paths:
       tags:
       - Workflow Definition
       x-accepts: application/json
+  /api/v1/workflowdefinitions/parameters/update/{valueProviderName}:
```

Comment:
I'm worried here. What happens if a workflow has the list of users retrieved from the database? Are we exposing those parameters to all of the workflows? Shouldn't we have a mechanism that does not allow it?

I'm a bit worried about this, because users maybe trust in the thing, and it's insecure by default.

Reply:
https://github.com/parodos-dev/parodos/pull/276/files#diff-087771281f17597fca047d56f3fd3e6e3ca16146e578817ca6ad2aa850245e13R18 the request to this controller can specify workName.  ValueProvider can use it to validate if the parameter is from a validated work in the workflow

Reply:
I'm not sure how that can fix a possible security issue :\

Reply:
ok, I will change the controller to WorkFlowDefinitionID/parameters/update/{valueProviderName} and use the workflowDefinition to validate the workflow requested.

Reply:
happy to track that on the jira, but keep the idea in your mind

Reply:
the controller has been updated to '/{workflowDefinitionName}/parameters/update/{valueProviderName}'

Reply:
the jira ticket is created: https://issues.redhat.com/browse/FLPATH-336